src/kit/Section.module.scss

```diff
@@ -0,0 +1,30 @@
+/* stylelint-disable no-descending-specificity */
+
+.base {
+  &:not(:last-child) {
+    padding-bottom: 16px;
+  }
+  .header {
+    border: 0 solid var(--theme-stage-border);
+    margin-bottom: 8px;
+  }
+  .title {
+    height: 32px;
+    line-height: 32px;
+    white-space: nowrap;
+  }
+  .title.string {
+    font-size: 16px;
+  }
+  .body {
+    height: max-content;
```

Comment:
q: isn't this basically the default behavior of html? a tag's height grows to fit its content unless specifically told not to.